.github/workflows/yarn.yml

```diff
@@ -1,23 +1,30 @@
 # This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Node.js CI
+name: Yarn CI & CD
 
 on:
-  push:
-    branches: [ main ]
+  # runs on all pull requests against main
```

Comment:
The intend of the next 2 lines seems clear enough to me that we can skip the comment

Reply:
Fixed in latest commit!